input/cql/HospitalHarmOpioidRelatedAdverseEventsFHIR.cql

```diff
@@ -0,0 +1,90 @@
+library HospitalHarmOpioidRelatedAdverseEventsFHIR version '0.1.000'
+
+using QICore version '4.1.1'
+
+include SupplementalDataElements version '3.4.000' called SDE
+include CQMCommon version '2.0.000' called CQMCommon
+include QICoreCommon version '2.0.000' called QICoreCommon
+include FHIRHelpers version '4.3.000' called FHIRHelpers
+
+codesystem "SNOMEDCT": 'http://snomed.info/sct'
+codesystem "HSLOC": 'http://cts.nlm.nih.gov/fhir/cs/hsloc'
+
+valueset "Emergency Department Visit": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.117.1.7.1.292'
+valueset "Encounter Inpatient": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.666.5.307'
+valueset "Observation Services": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1111.143'
+valueset "Operating Room Suite": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1248.141'
+valueset "Opioid Antagonist": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1248.119'
+valueset "Opioids, All": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1196.226'
+valueset "Routes of Administration for Opioid Antagonists": 'http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1248.187'
+
+code "Dead": '419099009' from "SNOMEDCT" display 'Dead'
+
+parameter "Measurement Period" Interval<DateTime>
+  default Interval[@2024-01-01T00:00:00.0, @2025-01-01T00:00:00.0)
```

Comment:
period should be from 2025-01-01 to 2025-12-31, this one is set to 2024 not 2025